### ISSUE TITLE
Problem: users cannot comment on issues

### ISSUE THREAD

**Issue by gazhayes** [EmurgoHK/Emurgis#32]
Problem: users can't comment on issues.

**Comment by AshHimself** [comment]
Claim

**Comment by samuelralak** [comment]
@AshHimself SimpleSchema will check for the existence of the defines keys rather than check for empty strings, so as long as you are - for instance - passing `comment` in 
```
postComment.call({
      problemId: problemId,
      comment: commentValue
}
```
the validation will always pass even if your input value is empty

**Comment by AshHimself** [comment]
Of course... makes sense.. i'll checkout blockrazor codebase on how implemented that functionality instead of reinvesting the wheel :)
